microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,379 @@ ms.author: msedgedevrel
+.NET (WinForms):
+* [CreationProperties Property](/dotnet/api/microsoft.web.webview2.winforms.webview2.creationproperties)
+* [CoreWebView2CreationProperties Class](/dotnet/api/microsoft.web.webview2.winforms.corewebview2creationproperties)
 
+WinRT:
+* [CoreWebView2Environment.CreateWithOptionsAsync Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createwithoptionsasync)
 
-<!-- 1. Calling a function ============================================================== -->
-## Calling a function
+Win32:
+* [CreateCoreWebView2EnvironmentWithOptions](/microsoft-edge/webview2/reference/win32/webview2-idl?#createcorewebview2environmentwithoptions)
 
-If you want to make your application use a specific runtime by calling a function, complete the following steps.
 
-### Win32\/C++
+##### [Registry key](#tab/registry-key)
 
-We'll use the [WebView2APISample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample) to demonstrate this procedure.
+1. Open a PowerShell terminal or PowerShell-enabled command prompt.
 
-1. On your development machine, find the path containing the Microsoft Edge preview channel.  For example:
+1. Modify and then run the following command:
 
-   `C:\\Users\\myname\\AppData\\Local\\Microsoft\\Edge SxS\\Application\\93.0.929.0`
+   ```reg
+   REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /v WebView2APISample.exe /t REG_SZ /d "C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0"
+   ```
 
-1. Clone the [WebView2Samples](https://github.com/MicrosoftEdge/WebView2Samples) repo.
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID. Replace the sample path with the path to the desired Microsoft Edge preview channel.
 
-1. Open the **WebView2APISample** project, and then in **Source Files**, open the `AppWindow.cpp` file.
 
-1. Find where [CreateCoreWebView2EnvironmentWithOptions](/microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environmentwithoptions) is called.  For example:
+##### [Environment variable](#tab/environment-variable)
 
-   ```cpp
-   HRESULT hr = CreateCoreWebView2EnvironmentWithOptions(
-       subFolder, m_userDataFolder.c_str(), options.Get(),
-       Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
-           this, &AppWindow::OnCreateEnvironmentCompleted)
-           .Get());
-   ```
+Name: `WEBVIEW2_BROWSER_EXECUTABLE_FOLDER`  
+Value: `<path to desired Microsoft Edge preview channel>`
 
-1. Replace the `subFolder` variable by the folder path for the Microsoft Edge preview channel you want to use.  For example:
+The environment variable will be applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per app.
```

Comment:
```suggestion
The target app can't be set when using the environment variable approach.  Therefore, when set as a global environment, this setting will affect all apps that use WebView2 on the machine.
```
This comment is a fresh entering of Nic's Outdated [comment](https://github.com/MicrosoftDocs/edge-developer/pull/3053/files/ebb0590c0e011b7e4455bacd633032af0600358e#r1561815133), where Nic asked if it would be more accurate to say it this way.  @champnic @victorhuangwq

Reply:
Rendered articles for review: 
* **Test upcoming APIs and features** (update to use new API)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-3053
* **Prerelease testing** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing?branch=pr-en-us-3053
* **Self-hosting** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/self-hosting?branch=pr-en-us-3053

For "Outdated" comments, to see latest Markdown/commits, use fresh Files Changed tab:
https://github.com/MicrosoftDocs/edge-developer/pull/3053/files